precompiles/peaq-rbac/src/selectors.rs

```diff
@@ -0,0 +1,46 @@
+use precompile_utils::keccak256;
+
+pub const SELECTOR_LOG_ADD_ROLE: [u8; 32] = keccak256!("RoleAdded(address,bytes32,bytes)");
```

Comment:
Seems to be the RBAC-ID-type, right? Especially for that, I defined that type within the peaq-primitives-xcm, have a look here: https://github.com/peaqnetwork/peaq-network-node/blob/185eff9e89db10221e38363f438ba761f61cbc7a/primitives/xcm/src/lib.rs#L87

Reply:
Correct! But we're just hashing some string so I didn't think we needed to have a placeholder type.